lib/src/input/figma/entities/layers/rectangle.dart

```diff
@@ -26,7 +24,6 @@ class FigmaRectangle extends FigmaVector
     type,
     pluginData,
     sharedPluginData,
-    FigmaStyle style,
     layoutAlign,
     FigmaConstraints constraints,
     boundaryRectangle,
```

Comment:
Should we remove the stoke-related attributes here? I see that they're also being passed to `super` 🤔

Reply:
That is correct